scripts/globals/summon.lua

```diff
@@ -278,7 +278,15 @@ xi.summon.avatarFinalAdjustments = function(dmg, mob, skill, target, skilltype,
 
     -- set message to damage
     -- this is for AoE because its only set once
-    skill:setMsg(xi.msg.basic.DAMAGE)
+    if
+        mob:getCurrentAction() == xi.action.PET_MOBABILITY_FINISH
+    then
```

Comment:
Theres only one condition here. It can be one line

Reply:
Done